### PR TITLE
Apply black style to test_GenBank.py, version 19.10b0; after  v19.3b0 was applied.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to test_GenBank.py; small changes as black v19.3b0 was already applied. Should be fine to merge.

### CODE REVIEW

File: Tests/test_GenBank.py [hunk 1]
```diff
@@ -1519,9 +1519,7 @@ def test_record_parser_13(self):
             record = next(records)
             length = 2007
             locus = "AB000048"
-            definition = (
-                "Feline panleukopenia virus DNA for nonstructural protein 1, complete cds"
-            )
+            definition = "Feline panleukopenia virus DNA for nonstructural protein 1, complete cds"
```
Comment (MarkusPiotrowski) [inline Tests/test_GenBank.py]: Isn't this strange? This doesn't fit in one line, does it? See the similar but shorter string in line 1600/1596, which is put on an extra line.

File: Tests/test_GenBank.py [hunk 2]
```diff
@@ -1597,7 +1593,9 @@ def test_record_parser_13(self):
             record = next(records)
             length = 1755
             locus = "AB000050"
-            definition = "Feline panleukopenia virus DNA for capsid protein 2, complete cds"
```
Comment (MarkusPiotrowski) [inline Tests/test_GenBank.py]: If this doesn't fit on the same line, how can line 1522 fit in one line?

File: Tests/test_GenBank.py (hunk 1 above)
Comment (peterjc) [reply Tests/test_GenBank.py]: This gives a line of 99, the old version had three lines the longest of which was 90. So both break black's target of 88, not much to choose between them (and different versions of black differed on this). I prefer the one line version.

We ought to break the string in two, I didn't notice in review,

File: Tests/test_GenBank.py (hunk 2 above)
Comment (peterjc) [reply Tests/test_GenBank.py]: The old version had one line, length 92 (too long).

The new version has three lines, the middle one is 83 long.

So this does make sense - but I agree the example above is different.

File: Tests/test_GenBank.py (hunk 1 above)
Comment (svalqui) [reply Tests/test_GenBank.py]: I'm not sure why black has removed the parentheses here and not on 1596.
usually if there is few PEP8 (line too long), I'll break them but this file have many so I have left that, I think that should be part of other Issue (PEP Compliance)?.

Comment (MarkusPiotrowski) [reply Tests/test_GenBank.py]: I was just curious about the 'different' behavior of black, but didn't realize that in this case, both versions produce too long lines. The PR is already merged, so I wouldn't re-open it.

Comment (peterjc) [reply Tests/test_GenBank.py]: Agreed - there is still an overly long line (due to the unbroken string), but there are a *lot* of overly long lines in the test suite. Something for another day.

**Pull request merged**